docs/manuals/core/concepts/scenario.md

```diff
@@ -1,17 +1,14 @@
-We introduce here the important concept of a `Scenario^`. A Taipy Scenario represents an instance of a
-business problem to solve on consistent data and parameter sets.
+Introducing the `Scenario`, a fundamental key concept in Taipy.
 
-As its name implies, with Taipy scenarios, the users can instantiate different versions of a business
-problem with different assumptions. This is extremely useful in a business context where impact analysis
-and what-if analysis are essential in the decision process.
+A Taipy scenario represents a business problem with consistent data and parameters.
```

Comment:
I would remove 'Taipy' here. Maybe use italics for the first occurrence of the term 'scenario'.